sadu-mapper/src/main/java/de/chojo/sadu/wrapper/util/Row.java

```diff
@@ -28,10 +28,7 @@
 import java.sql.SQLXML;
 import java.sql.Time;
 import java.sql.Timestamp;
-import java.time.LocalDate;
-import java.time.LocalDateTime;
-import java.time.LocalTime;
-import java.time.ZonedDateTime;
+import java.time.*;
```

Comment:
Probably good to keep it dedicated and not a star import.

Reply:
yes pls

Reply:
Done.